src/components/editor/Homepage.jsx

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Droppable, Draggable } from 'react-beautiful-dnd';
 import styles from '../../styles/App.module.scss';
+import elementStyles from '../../styles/isomer-cms/Elements.module.scss';
+import FormField from '../FormField';
 
```

Comment:
We should rename this file - `Homepage.jsx` is confusing given how we named our other pages. For example, `Pages` provides the links to each page, and clicking through brings you to `EditPage`, but `Homepage` does not perform the same function for `EditHomepage`. It is more of a collection of homepage elements / attributes / fields and this should be reflected as such in the file name.

Reply:
What do you think is a good name? Would `HomepageEditor` be less confusing?

Reply:
We could rename the `components/editor` folder to `components/homepage` and break `Homepage.jsx` up into separate files (for e.g. `HomepageResources`, `HomepageInfobar` etc.)